Makefile

```diff
@@ -6,6 +6,15 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+dev:
+	go get github.com/axw/gocov/gocov
+	go get github.com/golang/lint/golint
+	go get github.com/mattn/goveralls
+	go get github.com/pierrre/gotestcover
+	go get github.com/stretchr/testify/assert
```

Comment:
@petemoore what trickery is going on???? :)  I thought line 37 should fetch all packages we import.